### PR TITLE
unshadow: add page

### PR DESCRIPTION
**Pull request by ldorigo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by ldorigo** [comment]
Note: there's only two examples (which are really the same) because it's an extremely simple utility that does only one thing. Still, I think it would be nice for it to have a page.

File: pages/linux/unshadow.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# unshadow
+
+> Utility provided by the package "john", to combine /etc/passwd and /etc/shadow into a file that can be processed by John the Ripper. More information: <https://www.openwall.com/john/>.
```
Comment (andrik) [inline pages/linux/unshadow.md:3]:
```suggestion
> Utility provided by John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords. More information: <https://www.openwall.com/john/>.
```

Comment (sbrl) [reply pages/linux/unshadow.md:3]: Perhaps the `More information` could be on the next line, like this @andrik?
```suggestion
> Utility provided by the John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords.
> More information: <https://www.openwall.com/john/>.
```

File: pages/linux/unshadow.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# unshadow
+
+> Utility provided by the John the Ripper project to obtain the traditional Unix password file if the system uses shadow passwords.
+> More information: <https://www.openwall.com/john/>.
+
+- Combine the /etc/shadow and /etc/passwd of the current system:
```
Comment (agnivade) [inline pages/linux/unshadow.md:6]: I think these 2 examples should be just one. We always use tokens for options in commands. The value inside them can be concrete or generic. I had made the same comment some time earlier too. This is what I'd suggest:

Combine any two shadow and password files:

`sudo unshadow {{/etc/passwd}} {{/etc/shadow}}`.

**Pull request merged**